notebooks/renv/activate.R

```diff
@@ -0,0 +1,1220 @@
+
+local({
+
+  # the requested version of renv
+  version <- "1.0.7"
```

Comment:
this file seems to be imported from some external source.  will we need to do anything to keep it synced/up-to-date?  i presume (while along side the `notebooks/renv.lock`) it should continue to "just work" until we need it to do something it cant already do?